app/templates/components/goal/goal-index.hbs

```diff
@@ -1 +1,22 @@
-{{yield}}
+{{#fbf-nav primaryButton="drawer" transparent=true bigHeader=true title="Goal X" logo=responsive.isLayout withTabs=true imageUrl="http://www.pixelstalk.net/wp-content/uploads/2016/04/Dark-owl-wallpapers-HD.jpg" as |section|}}
```

Comment:
Drawing the nav should be done in a separate component, with a computed property for the logo and the title. This component can then be renamed to something like `detail-view` and rendered into the container for the content within the separate component.